### ISSUE TITLE
[Suggestion] up conveyor into side conveyor cover

### ISSUE THREAD

**Issue by Biviho** [stfwi/engineers-decor#184]
hi! i made this suggestion a while ago in IE, that was denied for some oddly reason, however...

the idea is basically to add a cover like the curved sheetmetal chute, but rotated, witch would be pretty cool and more pleasent to the eyes.

i'll leave a link to my old suggestion for the nice images: [the suggestion](https://github.com/BluSunrize/ImmersiveEngineering/issues/4486)
Let me know what you think!

**Comment by stfwi** [comment]
Hi man, I like the idea of having more decorative features out in the field. Having that said - Conveyors are IE core functionality. 
In ED I add decoration and small no-game-changer-things, and like to prevent interfering with the IE core. Hence, ED functional blocks are mostly industrial versions of vanilla devices, little nice-to-have machines, and also some experiments that broke out of their cage.
In other words, I like Blu to have full control about how conveyors are designed and functioning. Anything else would be bad-modder-manner. So - I'll have to close this one, too ;;
